pitch/auto-transpose/module.ily

```diff
@@ -30,7 +30,8 @@
    (set! all-translation-properties (cons symbol all-translation-properties))
    symbol)
 % add context properties descriptions
-%   transpose-direction
+
+#(translator-property-description 'insertKeySignatures boolean? "When #t (default) autotranspose will insert key signatures. Set to #f to turn off key signature inserts.")
```

Comment:
Add line breaks as you've done for transposeDirection.

Reply:
Done